app.js

```diff
@@ -1,14 +1,40 @@
 var http = require('http');
 var fs = require('fs');
-var path = require('path')
+var path = require('path');
+var mysql = require('mysql');
 const PORT = 3000;
+
+
 http.createServer(function (request, response) {
+    //console.log(request.url);
     
-if (request.url === '/favicon.ico') {           //code to handle favicon.ico image
-    response.writeHead(200, {'Content-Type': 'image/x-icon'} );
-    response.end();
+    if (request.url === '/favicon.ico') {           //code to handle favicon.ico image
+        response.writeHead(200, {'Content-Type': 'image/x-icon'} );
+        response.end();
+        return;
     }
+    var con = mysql.createConnection({
```

Comment:
this will create a connection to the server everytime a request comes in, please move this code out of the callback